acquire/collector.py

```diff
@@ -341,6 +341,9 @@ def collect_path(
             path = self.target.fs.path(path)
 
         try:
+            # If a path does not exist, is_dir(), is_file() and is_symlink() will return False (and not raise an
+            # exception), so we need to explicitly check for this.
+            exists = self.target.fs.lexists(str(path))
```

Comment:
I think there's one issue here currently, where if `path` is already a `TargetPath` but from a different filesystem. That might be an issue here: https://github.com/fox-it/acquire/blob/main/acquire/acquire.py#L256

I think you said that `path.get().lexists()` unfortunately doesn't work reliably, so then I guess it unfortunately has to become something like `path._fs.lexists(str(path))`?

Reply:
Hmm, that won't work then. I'll temporarily use the `path._fs.lexists(str(path))` way then until the issues with `TargetPath.get()` and `FileEntry.lexists()` are resolved.